docs/src/architecture.md

```diff
@@ -1,52 +1,50 @@
 # Architecture
 The Polygon Miden Architecture decribes the concepts of how the participants of the network can interact with each other. The architecture reflects the design goal: We aim to build an Ethereum scaling solution that extends its feature set. We want to enable developers to build faster, safer and more private dApps. Rollups allow the creation of new design spaces while retaining the collateral security of Ethereum. This is where to innovate, whereas the base layer should provide stability and evolve slowly.
 
-We believe that to attract billions of users to Ethereum, we need to extend it on three dimensions: scalability, safety, and privacy. 
-
-The actor model is our inspiration for achieving concurrent, local state changes on a distributed system like a blockchain. In the actor model, actors play the role of little state machines, meaning each actor is responsible for their own state. Actors have inboxes to send and receive messages to communicate with other actors. Messages can be read asynchronously.
+The [Actor Model](https://en.wikipedia.org/wiki/Actor_model) is our inspiration for achieving concurrent, local state changes on a distributed system like a blockchain. In the actor model, actors play the role of little state machines, meaning each actor is responsible for their own state. Actors have inboxes to send and receive messages to communicate with other actors. Messages can be read asynchronously.
```

Comment:
potential nit: should it be "in a distributed system"? (rather than "on a distributed system").

Reply:
I think you are right, thanks!